docs/DeveloperGuide.md

```diff
@@ -1203,6 +1264,7 @@ keeping all the non-affected data safely.
 
 ## Glossary
 
+* **Reps** - The process of repeating an exercise. Often abbreviated to 'reps'.
```

Comment:
Should the title be **Repetitions**? Because saying Reps is abbreviated to 'reps' might not be explaining anything.